dcos/config.py

```diff
@@ -14,7 +15,11 @@ def mutable_load_from_path(path):
     """
 
     with util.open_file(path) as config_file:
-        return MutableToml(toml.loads(config_file.read()))
+        try:
+            return MutableToml(toml.loads(config_file.read()))
+        except Exception as e:
```

Comment:
When in general should we use `logger.exception`?  Can you add that to the review wiki?

I understand adding it if there's valuable info that isn't contained in the exception message, but that's not the case here.